### PR TITLE
Redis: add support for multi-az

### PR DESCRIPTION
**Pull request by zucchinidev**
[#184471692](https://www.pivotaltracker.com/story/show/184471692)

Which other branches should this be merged with (if any)?

None

### CODE REVIEW

File: upgrading.html.md.erb [hunk 1]
```diff
@@ -73,6 +73,14 @@ Because previously created instances cannot be modified to use encryption at res
 `at_rest_encryption_enabled` to `false` in existing plans to match service instances that have already
 been created.
 
+#### <a id="chng-cust-plns-rest-encry"></a> Set `multi_az_enabled` to `false`
```
Comment (fnaranjo-vmw) [inline upgrading.html.md.erb:76]: Replace:

`<a id="chng-cust-plns-rest-encry"></a>`

with:

`<a id="chng-cust-plns-multi-az"></a>`

File: reference/aws-redis.html.md.erb [hunk 2]
```diff
@@ -172,6 +172,20 @@ provision only:
       <td><code>true</code></td>
       <td>provision</td>
     </tr>
+    <tr>
+      <td><code>multi_az_enabled</code></td>
+      <td>Boolean</td>
+      <td>
+        Whether to enable Multi-AZ Support for the replication group. If true, `node_count` must be greater than 2.
```
Comment (fnaranjo-vmw) [inline reference/aws-redis.html.md.erb:179]: Replace:
> Whether to enable Multi-AZ Support for the replication group. If true, `node_count` must be greater than 2.

With:
> Whether to enable Multi-AZ Support for the replication group. If true, `node_count` must be greater than 1.

File: upgrading.html.md.erb (hunk 1 above)
Comment (zucchinidev) [reply upgrading.html.md.erb:76]: Hi @fnaranjo-vmw, the docs team changed it.
Thank you very much @RichardJJG

File: reference/aws-redis.html.md.erb (hunk 2 above)
Comment (zucchinidev) [reply reference/aws-redis.html.md.erb:179]: Done! Thank you very much @fnaranjo-vmw

**Pull request merged**